adafruit_character_lcd/character_lcd.py

```diff
@@ -30,6 +30,12 @@
 
 """
 
+try:
+    from typing import Union, Optional, List, Sequence
+    from circuitpython_typing import pwmio
```

Comment:
Make sure you add this to `requirements.txt`!

Reply:
Should they go in `requirements.text` or `optional_requirements.tx` as they are not used to run the module?

Reply:
`requirements.txt` as they need to be downloaded as part of the runtime for Blinka platforms

Reply:
OK thx